torch/ao/quantization/fx/prepare.py

```diff
@@ -700,8 +700,6 @@ def maybe_insert_observers_before_graph_output(
     # arbitrary data structures. There is always a single output, and
     # that output can have arbitrary nesting of values. List[int] is
     # not the right data type for this.
-    assert output_quantized_idxs == [0] or output_quantized_idxs == [], \
```

Comment:
wait, no change is required to enable this? I think we need to unpack the tuple etc. to get the index?